### PR TITLE
[GO-8798] PQT - Data sync is throwing error on Android Phone/TAB [undefined is not an object (evaluating 't.code')]

### PR DESCRIPTION
**Pull request by robertkongsvmx**
### Fix ?
Link to ticket: https://servicemax.atlassian.net/browse/GO-8798

### What has been done
- set both content-type and Content-Type so multipart/form-data type will be set correctly in native code
- 

### How to test
- 
- 

### Todo
- [ ] 
- [ ] 

### Notes
- 
-

### CODE REVIEW

File: polyfill/Fetch.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ class RNFetchBlobFetchPolyfill {
           promise = Blob.build(body).then((b) => {
             blobCache = b
             options.headers['Content-Type'] = 'multipart/form-data;boundary=' + b.multipartBoundary
+            options.headers['content-type'] = 'multipart/form-data;boundary=' + b.multipartBoundary
```
Comment (yuhui-zhao) [inline polyfill/Fetch.js]: will two content-type be ok?

Comment (skyforeverGE) [reply polyfill/Fetch.js]: @yuhui-zhao because the original lib messed up the `content-type` and `Content-Type`.

Comment (yuhui-zhao) [reply polyfill/Fetch.js]: I understand. The question probably is for the header users that they might get two values.

Comment (maxpowa) [reply polyfill/Fetch.js]: Native portion of the library will only send the lowercase version.

Comment (yuhui-zhao) [reply polyfill/Fetch.js]: got it. Thanks @maxpowa

**Pull request merged**